### PR TITLE
feat: Accessible <Radio> and <RadioGroup>

### PR DESCRIPTION
**Pull request by sebald**

### CODE REVIEW

File: docs/content/components/radio.mdx [hunk 1]
```diff
@@ -21,90 +32,50 @@ import { Radio } from '@marigold/components';
...
-| `error` (optional)        | `boolean` | `false`     |
-| `errorMessage` (optional) | `string`  |             |
+| Name               | Type      | Default     | Description                                                                                                        |
+| :----------------- | :-------- | :---------- | :----------------------------------------------------------------------------------------------------------------- |
+| `variant`          | `string`  |             | Use a different _variant_ from theme                                                                               |
+| `size`             | `string`  | `'level-1'` | Use a different _size_ from theme                                                                                  |
+| `error` (optional) | `boolean` | `false`     | If `true`, the checkbox is considered invalid and if set the `errorMessage` is shown instead of the `description`. |
+| ...                |           |             | Yes you can use all regular attributes of `input`!                                                                 |
 
 ## Examples
```
Comment (ti10le) [inline docs/content/components/radio.mdx:42]: Examples for description and errorMessage?

File: packages/components/src/Radio/RadioGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  const styles = useComponentStyles(
+    'RadioGroup',
+    { variant, size },
+    { parts: ['container', 'group'] }
+  );
+
+  return (
+    <Box {...radioGroupProps} css={styles.container}>
+      {props.label && (
+        <Label as="span" required={required} {...labelProps}>
```
Comment (ti10le) [inline packages/components/src/Radio/RadioGroup.tsx:81]: Why not using the FieldBase with description, error etc.?

**Review by ti10le — CHANGES_REQUESTED** [review]

File: docs/content/components/radio.mdx (hunk 1 above)
Comment (sebald) [reply docs/content/components/radio.mdx:42]: Not supported yet, figma has no designs for this. Also, not supported by the `react-aria` hooks :-/

File: packages/components/src/Radio/RadioGroup.tsx (hunk 2 above)
Comment (sebald) [reply packages/components/src/Radio/RadioGroup.tsx:81]: Because it doesn't have that (yet)

**Review by ti10le — APPROVED** [review]

**Pull request merged**